Tests/UserServiceTests/Validators/UserEmailPasswordValidatorTests.cs

```diff
@@ -0,0 +1,402 @@
+﻿using System;
+using UserService.Validators;
+using DTO;
+using FluentValidation;
+using NUnit.Framework;
+using System.Linq;
+using DTO.RestRequests;
+
+namespace UserServiceTests.Validators
+{
+    public class UserEmailPasswordValidatorTests
+    {
+        private IValidator<CreateUserRequest> validator = new CreateUserRequestValidator();
+
+        [Test]
+        public void AllFieldsEmpty()
+        {
+            CreateUserRequest user = new CreateUserRequest();
```

Comment:
Why not var? (and other initializations)

Reply:
fixed